docs/release-payload-controllers.md

```diff
@@ -0,0 +1,73 @@
+
+# release-payload-controller
+The `release-payload-controller` is a single binary that contains all the controllers that are responsible for handling all the intereactions with `ReleasePayload` objects. 
+
+## release-controller
+The `release-controller` is responsible for creating all `ReleasePayload` objects.
+
+## release-payload-status-controller
+```
+Writes `.status`
+```
+* Do we need this if there isn't anything that needs to be written initially when a `ReleasePayload` is created?
+
+## release-payload-created-controller
+```
+writes: .status.condition.type=PayloadCreated
+reads: release creation job
+```
+### pseudocode
+```
+if job is successful, then PayloadCreated=true
+else, PayloadCreated=false with status of the job 
```

Comment:
if no job is found and the current payloadcreated state is not true or false, this is unknown, right/